bcl/src/bc_cmwx1zzabz.c

```diff
@@ -10,8 +10,7 @@
 // Apply changes to the factory configuration
 const char *_init_commands[] = {  
```

Comment:
If used locally, it should `static` - all private global identifiers should use prefix `_bc_cmwx1zzabz_...`